fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/domain/ReverseChainedSearchParam.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.domain;
+
+import com.ibm.fhir.search.parameters.QueryParameter;
+
+/**
+ * A chained search parameter
+ */
+public class ReverseChainedSearchParam extends SearchParam {
+
+    /**
+     * Public constructor
+     * @param name
+     */
+    public ReverseChainedSearchParam(String rootResourceType, String name, QueryParameter queryParameter) {
+        super(rootResourceType, name, queryParameter);
+    }
+
+    @Override
+    public <T> T visit(T query, SearchQueryVisitor<T> visitor) {
+        return null;
+        // return visitor.reverseChainedSearch(getName());
+    }
```

Comment:
???? what is going on here

Reply:
no such thing as a reverse chained parameter...it's all handled by the ChainedSearchParameter. Will remove.